### ISSUE TITLE
openssl:x86-windows & x64-windows build failed on win7sp1 with vs2015 professional update3

### ISSUE THREAD

**Issue by principality** [Microsoft/vcpkg#913]
$ vcpkg install openssl
-- CURRENT_INSTALLED_DIR=D:/project/c++/vcpkg/installed/x86-windows
-- DOWNLOADS=D:/project/c++/vcpkg/downloads
-- CURRENT_PACKAGES_DIR=D:/project/c++/vcpkg/packages/openssl_x86-windows
-- CURRENT_BUILDTREES_DIR=D:/project/c++/vcpkg/buildtrees/openssl
-- CURRENT_PORT_DIR=d:/project/c++/vcpkg/ports/openssl/.
-- Using cached D:/project/c++/vcpkg/downloads/openssl-1.0.2k.tar.gz
-- Testing integrity of cached file...
-- Testing integrity of cached file... OK
-- Extracting done
-- Applying patch D:/project/c++/vcpkg/ports/openssl/PerlScriptSpaceInPathFixes.patch
-- Applying patch failed. This is expected if this patch was previously applied.
-- Applying patch D:/project/c++/vcpkg/ports/openssl/PerlScriptSpaceInPathFixes.patch done
-- Applying patch D:/project/c++/vcpkg/ports/openssl/ConfigureIncludeQuotesFix.patch
-- Applying patch failed. This is expected if this patch was previously applied.
-- Applying patch D:/project/c++/vcpkg/ports/openssl/ConfigureIncludeQuotesFix.patch done
-- Applying patch D:/project/c++/vcpkg/ports/openssl/STRINGIFYPatch.patch
-- Applying patch failed. This is expected if this patch was previously applied.
-- Applying patch D:/project/c++/vcpkg/ports/openssl/STRINGIFYPatch.patch done
-- Build x86-windows-rel
-- Build x86-windows-rel done
-- Build x86-windows-dbg
-- Build x86-windows-dbg done
-- Installing: D:/project/c++/vcpkg/packages/openssl_x86-windows/share/openssl/copyright
-- Performing post-build validation
The folder /include is empty. This indicates the library was not correctly installed.
Found 1 error(s). Please correct the portfile:
    d:\project\c++\vcpkg\ports\openssl\portfile.cmake
-- Performing post-build validation done
Error: Building package openssl:x86-windows failed with: POST_BUILD_CHECKS_FAILED
Please ensure you're using the latest portfiles with `.\vcpkg update`, then
submit an issue at https://github.com/Microsoft/vcpkg/issues including:
  Package: openssl:x86-windows
  Vcpkg version: 0.0.79-2e7896e89e7f0cb15b30b7d087f8001c57b3220a

BTW. I installed openssl on win10 with vs2015 successfully.

**Comment by ras0219-msft** [comment]
Hi @principality, thanks for reporting this!

Could you attach the following logs from `buildtrees\openssl\`?
- build-x86-windows-rel-out.log
- build-x86-windows-rel-err.log
- configure-perl-x86-windows--rel-err.log
- configure-perl-x86-windows--dbg-err.log
- configure-do-x86-windows--rel-err.log
- configure-do-x86-windows--dbg-err.log

If you'd prefer to send them to vcpkg@microsoft.com, that works too!

**Comment by principality** [comment]
I checked and found configure-perl-x86-windows--rel-err.log & configure-perl-x86-windows-dbg-err.log is empty.  build-x86-windows-rel-out.log & -err.log does not contain any useful information. It seems that configure-do reports some useful information. Please check the uploaded files. Thank you. @ras0219-msft 

[build-x86-windows-rel-err.log.txt](https://github.com/Microsoft/vcpkg/files/907319/build-x86-windows-rel-err.log.txt)
[build-x86-windows-rel-out.log.txt](https://github.com/Microsoft/vcpkg/files/907320/build-x86-windows-rel-out.log.txt)

[configure-do-x86-windows--dbg-err.log.txt](https://github.com/Microsoft/vcpkg/files/907317/configure-do-x86-windows--dbg-err.log.txt)
[configure-do-x86-windows--rel-err.log.txt](https://github.com/Microsoft/vcpkg/files/907318/configure-do-x86-windows--rel-err.log.txt)

**Comment by ras0219-msft** [comment]
Ok, it looks like something messed up during the configure step. For reference, my `configure-do-x86-windows--rel-err.log.txt` is simply:
```
unknown option - enable-capieng
unknown option - enable-capieng
```

As a hunch, I suspected it would be an issue with having `+`'s in your path, but I tried renaming the directory locally and your issue didn't repro.

Could you attach or send the `configure-do-x86-windows--rel-out.log.txt`?

**Comment by principality** [comment]
@ras0219-msft I have resolved the problem. By copying the perl module "warnings/register.pm" to "\downloads\tools\perl\perl\lib\" from ActivePerl, everything works well. 

I attached the configure-do-x86-windows--rel-out.log. But it was generated when I finished the installation. Does this file help? If neccessary, I can repeat the issue and re-generate the log file.
[configure-do-x86-windows--rel-out.log.txt](https://github.com/Microsoft/vcpkg/files/913868/configure-do-x86-windows--rel-out.log.txt)
